de/cheatsheet-deep-learning.md

```diff
@@ -1,162 +1,162 @@
 **1. Deep Learning cheatsheet**
 
-&#10230;
+&#10230; Deep Learning Spickzettel
 
 <br>
 
 **2. Neural Networks**
 
-&#10230;
+&#10230; Neuronale Netze
 
 <br>
 
 **3. Neural networks are a class of models that are built with layers. Commonly used types of neural networks include convolutional and recurrent neural networks.**
 
-&#10230;
+&#10230; Neuronale Netze sind eine Klasse von Modellen die in Schichten aufgebaut sind. Gängige Typen neuronaler Netze sind unter anderem faltende und rekurrente neuronale Netze.
 
 <br>
 
 **4. Architecture ― The vocabulary around neural networks architectures is described in the figure below:**
 
-&#10230;
+&#10230; Architektur - Das Vokabular rund um Architekturen neuronaler Netze ist in folgender Abbildung beschrieben: 
 
 <br>
 
 **5. [Input layer, hidden layer, output layer]**
 
-&#10230;
+&#10230; [Eingangsschicht, versteckte Schicht, Ausgangsschicht]
 
 <br>
 
 **6. By noting i the ith layer of the network and j the jth hidden unit of the layer, we have:**
 
-&#10230;
+&#10230; Sei i die i-te Schicht des Netzes und j die j-te versteckte Einheit der Schicht, so ist:
 
 <br>
 
 **7. where we note w, b, z the weight, bias and output respectively.**
 
-&#10230;
+&#10230; wobei w, b und z jeweils Gewichtung, Vorspannung und Ergebnis bezeichnen.
```

Comment:
Bias = _Verzerrung_ instead of "Vorspannung"
weight = _Gewicht_ instead of "Gewichtung"
output = _Ausgabe_ instead of "Ergebnis"
[source](https://www-ai.cs.uni-dortmund.de/LEHRE/VORLESUNGEN/MLRN/SKRIPT/MLVgesamt.de.article.pdf)

just a suggestion, so the words resemble current literature